tensorflow/core/kernels/conv_grad_filter_ops.cc

```diff
@@ -980,6 +992,106 @@ void LaunchConv2DBackpropFilterOp<Eigen::GpuDevice, T>::operator()(
   // if we do not have a cached algorithm_config for this conv_parameters
   cudnn_use_autotune = true;
 #endif
+#if GOOGLE_CUDA && CUDNN_VERSION >= 8100
```

Comment:
Can you use runtime dispatch? We can let StreamExecutor return a special error (e.g. UnimplementedError) and switch on it at runtime.

Reply:
Done.